src/main/scripts/src/app/shared/table/model/table-row-cell.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * -----------------------------------------------------------------------\
+ * Lumeer
+ *
+ * Copyright (C) since 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------/
+ */
+
+export interface TableRowCell {
+
+  label: string;
+  active: boolean;
+  hidden: boolean;
+  constraints: string[];
```

Comment:
Because we have lines as separate component. So while editing value on some cell, we don't have access to header (which holds constraints).
Same with `hidden`, each cell has it's own css class.